pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[tool.codespell]
+ignore-words-list = [
+    "linz",
+    "socio-economic",
+    "therefrom",
+]
+skip = [
+    "docs/clay-v0/tutorial_digital_earth_pacific_patch_level.ipynb",
+    "docs/clay-v0/partial-inputs.ipynb",
+    "docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb",
+]
```

Comment:
Codespell catches some false positive misspellings in Jupyter Notebook binary outputs (see also https://github.com/codespell-project/codespell/issues/2138), so skipping these files after the true positive misspellings have been fixed:

```
docs/clay-v0/tutorial_digital_earth_pacific_patch_level.ipynb:245: iNH ==> in
docs/clay-v0/tutorial_digital_earth_pacific_patch_level.ipynb:687: te ==> the, be, we, to
docs/clay-v0/tutorial_digital_earth_pacific_patch_level.ipynb:916: WEe ==> we
docs/clay-v0/tutorial_digital_earth_pacific_patch_level.ipynb:985: Nd ==> And, 2nd
docs/clay-v0/tutorial_digital_earth_pacific_patch_level.ipynb:985: FO ==> OF, FOR, TO, DO, GO
docs/clay-v0/tutorial_digital_earth_pacific_patch_level.ipynb:985: OT ==> TO, OF, OR, NOT, IT
docs/clay-v0/tutorial_digital_earth_pacific_patch_level.ipynb:985: bu ==> by, be, but, bug, bun, bud, buy, bum
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: oly ==> only
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: teH ==> the
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: FO ==> OF, FOR, TO, DO, GO
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: tRU ==> through, true
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: AAs ==> ass, as
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: ALo ==> also
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: ALo ==> also
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: ALo ==> also
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: ALo ==> also
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: ALo ==> also
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: ALo ==> also
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: ALo ==> also
docs/tutorials/v1-inference-simsearch-naip-stacchip.ipynb:708: ALo ==> also
docs/clay-v0/partial-inputs.ipynb:394: oT ==> to, of, or, not, it
docs/clay-v0/partial-inputs.ipynb:394: fPr ==> for, far, fps
```